pkg/scheduler/framework/runtime/framework.go

```diff
@@ -60,6 +60,16 @@ const (
 	permit                      = "Permit"
 )
 
+var allClusterEvents = []framework.ClusterEvent{
```

Comment:
I think this list should be defined in the same place where the event handlers are set up to make sure they are in sync. WDYT?

Reply:
Hmm... I may choose to leave them here as otherwise, the runtime package needs to depend on pkg/scheduler (eventhandlers.go).

Reply:
sg, I think eventually we should put them in their own `events` pkg to break those dependencies.